libbeat/logp/metrics.go

```diff
@@ -56,6 +56,23 @@ func snapshotMetrics() monitoring.FlatSnapshot {
 	return monitoring.CollectFlatSnapshot(monitoring.Default, monitoring.Full, true)
 }
 
+// List of metrics that are gauges, so that we know for which to
+// _not_ subtract the previous value in the output.
+// TODO: Replace this with a proper solution that uses the metric
+// type from where it is defined. See:
+// https://github.com/elastic/beats/issues/5433
+var gauges = map[string]bool{
+	"libbeat.pipeline.events.active": true,
+	"libbeat.pipeline.clients":       true,
+	"libbeat.config.module.running":  true,
+	"registrar.states.current":       true,
+	"filebeat.harvester.running":     true,
+	"filebeat.harvester.open_files":  true,
+	"beat.memstats.memory_total":     true,
+	"beat.memstats.memory_alloc":     true,
+	"beat.memstats.gc_next":          true,
+}
```

Comment:
[This one](https://github.com/elastic/beats/blob/33270f1305dec9f286bcd3a6f91daac252d9ab21/winlogbeat/eventlog/cache.go#L54) presents a problem because the name is not known ahead of time. Perhaps a pattern could be used to handle it or we just ignore it (I doubt anyone will be using this metric before we have a final solution).

Reply:
Yeah, I saw that one. I'd say we just ignore it in this phase and it can be handled with the real solution.